src/libraries/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str_str.cs

```diff
@@ -724,19 +724,19 @@ public void SearchPatternCaseInsensitive()
         {
             DirectoryInfo testDir = Directory.CreateDirectory(GetTestFilePath());
             string testBase = GetTestFileName();
-            testDir.CreateSubdirectory(testBase + "yZZz");
-            testDir.CreateSubdirectory(testBase + "yZZZ");
+            testDir.CreateSubdirectory(testBase + "yZZZZZZZZZZz");
```

Comment:
Are these changes improving something in particular?

Reply:
Yes without this the tests would sometimes discover extra files because the random names may now contain eg "zz"